worker/include/RTC/RTCP/CompoundPacket.h

```diff
@@ -12,21 +12,17 @@ namespace RTC { namespace RTCP
 	class CompoundPacket
 	{
 	public:
-		~CompoundPacket();
 
-		uint8_t* GetRaw();
```

Comment:
In RTCP packet implementation the `raw` attribute holds an internal buffer and `GetRaw` returns a pointer to it. This buffer is allocated when calling `Serialize()` (with no argument).

In `CompoundPacket` there is no reason for having an internal buffer, so `raw` attribute will point to the buffer provided in `Serialize(uint8_t buffer)`, Ok?